src/client.rs

```diff
@@ -268,7 +270,7 @@ impl Pulsar {
             })
     }
 
-    pub fn create_producer<S: Into<String>>(
+    pub fn create_producer<S: Into<String>, E: PulsarExecutor>(
```

Comment:
I'm not sure I'm understanding why the methods that don't themselves take a `E: PulsarExecutor` need the `E` type param. Can you clarify? I'm referring to `Pulsar::create_producer`, `Pulsar::create_partitioned_producers`, `TopicProducer::from_connection`, `Producer::send`, and `Producer::send_raw`

Reply:
I think you're right, I may have confused them with `ConnectionManager::from_connection`. Let me try to get rid of them and update the PR.

Reply:
I've rebased from master and fixed these, you're right they are unnecessary.